	Adding nVidia Cuda Toolkit 11.8
	Adding nVidia cuDNN 8.6 (.0.163)
WARNING (pytensor.tensor.blas): Using NumPy C-API based implementation for BLAS functions.
TypeError: float() argument must be a string or a number, not 'TensorVariable'

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "main.py", line 81, in <module>
    trace = posterior_inference_mcmc(Xu_noise, jnp.eye(2)*prior_var, param_iter, Xu_fixed, Xf, Y)  #TODO: Y
  File "/local/236165/include/mcmc_posterior.py", line 36, in posterior_inference_mcmc
    z_uncertain = jnp.array(z_uncertain_list)
  File "/home/yanw/.local/lib/python3.8/site-packages/jax/_src/numpy/lax_numpy.py", line 2032, in array
    out = np.array(object, dtype=dtype, ndmin=ndmin, copy=False)
ValueError: setting an array element with a sequence.
--------------------------------------------------------------------------
Primary job  terminated normally, but 1 process returned
a non-zero exit code. Per user-direction, the job has been aborted.
--------------------------------------------------------------------------
--------------------------------------------------------------------------
mpirun detected that one or more processes exited with non-zero status, thus causing
the job to be terminated. The first process to do so was:

  Process name: [[23982,1],0]
  Exit code:    1
--------------------------------------------------------------------------
cp: missing destination file operand after '*.png/home/yanw/uncertain/results/figures/0920'
Try 'cp --help' for more information.
cp: missing destination file operand after '/home/yanw/uncertain'
Try 'cp --help' for more information.
